.github/workflows/ci.yml

```diff
@@ -0,0 +1,163 @@
+name: CI
+
+on:
+  release:
+    types: [created]
+  push:
+  schedule:
+    # Runs every Thursday at 20:23 GMT to avoid bit rod
+    - cron: "20 23 * * 4"
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        exclude:
+          # There is a known issue where Python C extensions
+          # can not be built on Python 3.5 on Windows.
+          # https://github.com/actions/virtual-environments/issues/34
+          # Therefore, ujson can't be compiled and the build would fail.
+          - os: windows-latest
+            python-version: 3.5
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install latest nightly
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          override: true
+
+      - name: Lint with rustfmt
+        uses: actions-rs/cargo@v1
+        with:
+          command: fmt
+
+      - name: Lint with clippy
+        uses: actions-rs/cargo@v1
+        with:
+          command: clippy
+          args: --all-targets --all-features
+
+      - name: Test with cargo
+        uses: actions-rs/cargo@v1.0.1
+        with:
+          command: test
+          toolchain: nightly
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install poetry
```

Comment:
Not sure if this applicable/reasonable here, but poetry recommends using `curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python` while noting for pip "Be aware, however, that it will also install poetry's dependencies which might cause conflicts." (https://github.com/python-poetry/poetry#installation).

Reply:
Hum... good point. Maybe we should also use that: https://github.com/marketplace/actions/python-poetry-action

Reply:
Decided to keep this for now until we run into problems with the build.
The reason is, that I don't want to add another dependency to a third-party action and that the pip command is a bit easier on the eye. 👀